proto/config.proto

```diff
@@ -151,30 +172,41 @@ message ListImageTemplatesResponse {
   repeated ImageTemplate image_templates = 1;
 }
 
+message ListAllImageTemplatesRequest {
```

Comment:
I also made the changes to the Image template, but let me know if this is not necessary.

Reply:
This should be ok

Reply:
I think this is ok